Podcast/BookmarkTableViewCell.swift

```diff
@@ -157,7 +157,7 @@ class BookmarkTableViewCell: UITableViewCell {
         separator.frame = CGRect(x: 0, y: frame.height - separatorHeight, width: frame.width, height: separatorHeight)
     }
     
-    func setupWithEpisode(episode: Episode) {
+    func setup(with episode: Episode, downloadStatus: DownloadStatus) {
```

Comment:
why do you take in `downloadStatus` if you don't use it in the function?

Reply:
In case we ever want a little downloaded icon on here. I'm sure we will eventually. This will change soon anyway with my upcoming PRs.